.github/workflows/validate-pull-request.yml

```diff
@@ -0,0 +1,9 @@
+name: Validate Pull Request
+
+on:
+  pull_request:
+
+jobs:
+  validate-pull-request:
```

Comment:
Configure this for the actual parent repository since it fails now: https://github.com/Lombiq/GitHub-Actions/blob/dev/Docs/Workflows/Productivity/ValidateSubmodule.md